NrqlAlertCondition/src/test/java/com/newrelic/alerts/nrqlalert/CreateHandlerTest.java

```diff
@@ -76,25 +77,28 @@ public void handleRequest_SimpleSuccess() throws IOException, AlertApiException
         nrqlObject.put("query", "SELECT count(*) from AwsLambdaInvocationError");
         nrqlObject.put("since_value", "3");
 
-        when(alertApiClient.create(any(NewRelicNrqlCondition.class), any(String.class), any(Integer.class))).thenReturn(jsonObject);
+        when(alertApiClient.create(
+                        any(NewRelicNrqlCondition.class), any(String.class), any(Integer.class)))
+                .thenReturn(jsonObject);
 
-        final ResourceHandlerRequest<ResourceModel> request = ResourceHandlerRequest.<ResourceModel>builder()
-                .desiredResourceState(model)
-                .build();
+        final ResourceHandlerRequest<ResourceModel> request =
+                ResourceHandlerRequest.<ResourceModel>builder().desiredResourceState(model).build();
 
-        final ProgressEvent<ResourceModel, CallbackContext> response
-                = createHandler.handleRequest(proxy, request, null, logger);
+        final ProgressEvent<ResourceModel, CallbackContext> response =
+                createHandler.handleRequest(proxy, request, null, logger);
 
         assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
 
-        // verify that the model was actually updated with returned data (see json object data above)
+        // verify that the model was actually updated with returned data (see json object data
+        // above)
         NrqlCondition afterCondition = this.model.getNrqlCondition();
         assertThat(afterCondition.getTerms().isEmpty()).isFalse();
         Term firstTerm = afterCondition.getTerms().get(0);
         assertThat(afterCondition.getId()).isEqualTo(1);
         assertThat(afterCondition.getType()).isEqualTo("NrqlAlert");
         assertThat(afterCondition.getEnabled()).isTrue();
-        assertThat(firstTerm.getThreshold()).isEqualTo("20");
+        assertThat(firstTerm.getDuration()).isEqualTo(1);
+        assertThat(firstTerm.getThreshold()).isEqualTo(20d);
```

Comment:
Is `20d` a valid value here?

Reply:
It is. It indicates a double-precision floating point literal. Double is the default for floats in Java, so most of the time, you'll just see 20.0.